common/constants.go

```diff
@@ -50,9 +50,11 @@ const (
 	NatsSyncClientConfigMapServiceAccountName = "natssync-client-configmap-serviceaccount"
 	NatsSyncClientConfigMapVolumeName         = "natssync-client-configmap-volume"
 
-	NeptuneName       = "neptune-controller-manager"
-	NeptuneDefaultTag = "e056f69"
-	NeptuneTagFile    = "common/neptune_manager_tag.txt"
+	NeptuneName                  = "neptune-controller-manager"
+	NeptuneDefaultTag            = "e056f69"
```

Comment:
Since this is a cherry pick, should the neptune default tag be something different than the original PR merged to main?